pkg/operator/target_config_reconciler.go

```diff
@@ -953,6 +954,14 @@ func (c *TargetConfigReconciler) manageDeployment(kueueoperator *kueuev1.Kueue,
 		},
 	}
 
+	required.Spec.Template.Spec.PriorityClassName = "system-cluster-critical"
+	required.Spec.Template.Spec.Containers[0].Resources = v1.ResourceRequirements{
+		Requests: v1.ResourceList{
+			v1.ResourceCPU:    resource.MustParse("500m"),
```

Comment:
@kannon92 I'm enforcing the same values from here https://github.com/openshift/kueue-operator/blob/main/bindata/assets/kueue-operator/deployment.yaml#L59-L61

Are those the expected values?

Reply:
I think that is fine.